WordPress/gutenberg
# escape-html: set up autogenerated API docs

**Related to** https://github.com/WordPress/gutenberg/issues/14227

This PR sets up auto-generated API docs for the `escape-html` package through the `npm run docs:build` command and updates its README file.

## Test instructions

- Delete or modify the contents at `packages/escape-html/README.md` within the start `` and end `` tokens.
- Execute `npm run docs:build` and verify they are re-created.
- Review that the API docs match the JSDoc of the exported symbols.
